### ISSUE TITLE
Bug GUI Workflow Requests Tickets.without tasknumbers

### ISSUE THREAD

**Issue by alf-cactus** [CactuseSecurity/firewall-orchestrator#2647]
In GUI Workflow Requests Tickets. If you create a ticket with tasks with API and passed no tasknumber:

The Ticket is written to database but breaks the GUI. No more tickets, even valid ones, are displayed.

Fix: remove the ticket via API

Solutions: Either make tasknumber mandatory (@alf-cactus could do that) or change GUI behavior

**Comment by abarz722** [comment]
As the task number is only used as a sort criterion, a default value of 0 would probably be sufficient for the beginning. I have several tickets with all task numbers 0 in my database without causing problems.

**Comment by tpurschke** [comment]
@abarz722 could we not simply drop the task_number and use the id of the reqtask table for sorting  assuming  tasks are created in the correct order?
So we would just have to generate the task number actually displayed (in the UI) - similar to the rule number.

**Comment by tpurschke** [comment]
> [@abarz722](https://github.com/abarz722) could we not simply drop the task_number and use the id of the reqtask table for sorting assuming tasks are created in the correct order? So we would just have to generate the task number actually displayed (in the UI) - similar to the rule number.

no, this approach would not allow for creating tasks in a specific order (e.g. in planning of modelling module)
